src/pages/TrueSightV2/components/TruesightFooter.tsx

```diff
@@ -1,8 +1,8 @@
 import { Trans } from '@lingui/macro'
-import { Text } from 'rebass'
+import { Flex, Text } from 'rebass'
 import styled, { keyframes } from 'styled-components'
 
-import { ButtonPrimary } from 'components/Button'
+import { ReactComponent as WalletIcon } from 'assets/svg/wallet_icon.svg'
```

Comment:
Can we use `src/components/Icons/Wallet.tsx` ?

Reply:
nice